mmv1/third_party/terraform/services/cloudfunctions2/resource_cloudfunctions2_function_test.go

```diff
@@ -125,6 +125,7 @@ resource "google_cloudfunctions2_function" "terraform-test2" {
         object = google_storage_bucket_object.object.name
       }
     }
+    on_deploy_update_policy {}
```

Comment:
Can you add an update test that starts with this set, then switches to the other type of update policy, then removes the update policy entirely?

Reply:
Still need this update test

Reply:
Working on the test now. Slowed down by production outage yesterday

Reply:
This proposed test case revealed additional issues in the backend implementation. I will add tests and update this PR once the fixes for these issues are rolled out to production.

Reply:
Ok whew. The feature is fully rolled out in the API. I added update tests as you described.